data/db.sql

```diff
@@ -16,4 +17,59 @@ INSERT INTO "user"(firstname, lastname, birthday) VALUES ('Vicky', 'Pearson', '1
 INSERT INTO "user"(firstname, lastname, birthday) VALUES ('Silvia', 'Mcguire', '1971-03-02');
 INSERT INTO "user"(firstname, lastname, birthday) VALUES ('Brendan', 'Pena', '1950-02-17');
 INSERT INTO "user"(firstname, lastname, birthday) VALUES ('Jackie', 'Cohen', '1967-01-27');
-INSERT INTO "user"(firstname, lastname, birthday) VALUES ('Delores', 'Williamson', '1961-07-19');
\ No newline at end of file
+INSERT INTO "user"(firstname, lastname, birthday) VALUES ('Delores', 'Williamson', '1961-07-19');
+*/
+
+
+CREATE TABLE Utilisateur(
+    id_user SERIAL NOT NULL PRIMARY KEY,
```

Comment:
Pas besoin du NOT NULL sur un SERIAL https://www.postgresql.org/docs/9.1/datatype-numeric.html